### ISSUE TITLE
17. Bonus: how to watch tests with Mocha

### ISSUE THREAD

**Issue by filipedeschamps** [filipedeschamps/rss-feed-emitter#137]
Like I said in the previous issue, from now on you will just keep iterating over the same things, which are:

* Continue to develop the implementation details
* Continue to write tests to cover your module

You need to keep doing them in **parallel**. What I do is open both the module and test code, and in the bottom, I keep watching the tests running automatically while I keep changing those files:

![image](https://cloud.githubusercontent.com/assets/4248081/13554998/ca2f7022-e394-11e5-903d-60ec44818d1d.png)

If I do something wrong or I'm in the middle of the development, it's very easy way to see once everything is ok.

![image](https://cloud.githubusercontent.com/assets/4248081/13555056/d90ee4c8-e395-11e5-8e7c-ac0eb52e3fdd.png)

## Watch script

This is very easy to do, just add this to your scripts property inside `package.json`:

``` js
"test-unit-watch": "mocha --watch --compilers js:babel-core/register test/unit/**/*.spec.js"
```

Basically we re just adding a `--watch` flag to Mocha.

## Next step
[18. Bonus: how to mock requests in Unit Tests](https://github.com/filipedeschamps/rss-feed-emitter/issues/138)
